docs/doc/30-reference/30-sql/20-query-syntax/dml-json-path.md

```diff
@@ -0,0 +1,75 @@
+---
+title: JSON PATH
+---
+
+Databend supports Semi-structured data type and allow retrieving the inner elements by JSON path operators:
```

Comment:
`Semi-structured data type` can add a link to the semi-structured data type markdown page.

Reply:
ok